docs/README.md

```diff
@@ -22,6 +22,18 @@ To run ImageLab from the command line:
   
  Note that on Windows platforms, the ":" character in the classpath must be changed to the ";" character.)
 
+ ### Run with BlueJ
```

Comment:
Let's make this wording consistent with the previous subsection, "To run ImageLab using BlueJ"